code/modules/admin/fun_verbs.dm

```diff
@@ -1001,27 +1001,40 @@
 	message_admins("[ADMIN_TPMONTY(IF)] started being imaginary friend of [ADMIN_TPMONTY(L)].")
 
 
-	
 /datum/admins/proc/force_dropship()
 	set category = "Fun"
 	set name = "Force Dropship"
 
 	if(!check_rights(R_FUN))
 		return
 
-	if(!length(SSshuttle.dropships))
+	if(!length(SSshuttle.dropships) && !SSshuttle.canterbury)
+		message_admins("no ships")
 		return
 
-	var/obj/docking_port/mobile/marine_dropship/D = SSshuttle.dropships[1]
-
-	if(!istype(D))
+	var/list/possible_destinations
+	var/list/available_shuttles = SSshuttle.dropships + list(SSshuttle.canterbury)
+	message_admins("availalbe shuttles [length(available_shuttles)]")
+	var/selection = input(usr, "Which shuttle do you want to move?", "Force Dropship") as null|anything in available_shuttles
+	if(!selection)
+		message_admins("no ship select")
```

Comment:
Still borked

Reply:
Try not creating additional lists, and like I said, add more debugging information, loop through the lists, examine their structure.